python/mxnet/contrib/quantization.py

```diff
@@ -40,7 +40,7 @@
 from ..module import Module
 
 
-def _quantize_params(qsym, params):
+def _quantize_params(qsym, params, th_dict):
```

Comment:
Does th_dict mean threashold_dict? I don't understand what th stands for

Reply:
I guess it means threashold_dict, @reminisce can you explain it?

Reply:
Yes, it means threshold.